src/internal/testutil/identity.go

```diff
@@ -23,7 +23,7 @@ const DexMockConnectorEmail = `kilgore@kilgore.trout`
 // OIDCOIDCConfig is an auth config which can be used to connect to the identity service in tests
 func OIDCOIDCConfig() *auth.OIDCConfig {
 	return &auth.OIDCConfig{
-		Issuer:          "http://localhost:30658/",
+		Issuer:          "http://pachd:1658/",
```

Comment:
This doesn't work in minikube by default because hairpin network routes (where a pod connects to a service it's a part of) don't work. So we may need both the re-write for localhost _and_ the issuer change for this to be effective :/